### PR TITLE
Feature/ADF-154/Apply taoCe rules to Item Class role

### PR DESCRIPTION
**Pull request by shpran**
**Relates to:** _[ADF-154](https://oat-sa.atlassian.net/browse/ADF-154)_

**Changes:**
* necessary rules applied to the new item class roles.

**Requires:**
* [oat-sa/tao-core#2883](https://github.com/oat-sa/tao-core/pull/2883)
* [oat-sa/extension-tao-item#466](https://github.com/oat-sa/extension-tao-item/pull/466)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,80 +1,80 @@
...
-		"oat-sa/extension-tao-itemqti" : ">=27.0.0",
-		"oat-sa/extension-tao-testtaker" : ">=8.0.0",
-		"oat-sa/extension-tao-testqti-previewer" : ">=3.0.0",
-		"oat-sa/extension-tao-test" : ">=15.0.0"
-	},
-    "autoload" : {
-        "psr-4" : {
-            "oat\\taoCe\\" : ""
+        "oat-sa/generis": ">=14.0.0",
+        "oat-sa/tao-core": ">=48.2.0",
```
Comment (gabrielfs7) [inline composer.json:60]: Do not forget to update tao-core to new version before merge :)

**Review by gabrielfs7 — APPROVED** [review]
LGTM, good job! 

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

File: manifest.php [hunk 2]
```diff
@@ -32,6 +35,16 @@
...
+        [
+            AccessRule::GRANT,
+            TaoItemsRoles::ITEM_CLASS_NAVIGATOR,
+            ['ext' => 'taoCe', 'mod' => 'Main', 'act' => 'index'],
+        ],
+        [
+            AccessRule::GRANT,
+            TaoItemsRoles::ITEM_CLASS_NAVIGATOR,
+            ['ext' => 'taoCe', 'mod' => 'Home'],
+        ],
```
Comment (boajer) [inline manifest.php:47]: Lame question, but is it possible we have new roles running without installing taoCe?

**Review by boajer — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Pull request closed without merging**